streaming/index.js

```diff
@@ -312,7 +293,8 @@ const startServer = async () => {
 
     if (subs[channel].length === 0) {
       log.verbose(`Subscribe ${channel}`);
-      redisSubscribeClient.subscribe(channel, onRedisMessage);
+      redisSubscribeClient.subscribe(channel)
+	  redisSubscribeClient.on("message", onRedisMessage);
```

Comment:
Does each subscribe require a "on message"? Or is this applying at client level? If so, you can lift this up to when you initialise the redis client for subscriptions, otherwise you'll be getting many invocations of the same callback each message.

Reply:
I think this should just be done once.

Reply:
I think you're right, it per client.